packages/paywall/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unlock-protocol/paywall",
-  "version": "0.6.0",
```

Comment:
where 0.6.1?

Reply:
we auto bump if version change is required and readme change bumped it up.